ext/crypto/x25519.rs

```diff
@@ -42,6 +42,17 @@ pub fn op_crypto_generate_x25519_keypair(
   pubkey.copy_from_slice(&x25519_dalek::x25519(pkey, X25519_BASEPOINT_BYTES));
 }
 
+#[op2]
+#[string]
+pub fn op_crypto_x25519_public_key(#[buffer] private_key: &[u8]) -> String {
+  use base64::Engine;
+
+  let private_key: [u8; 32] =
+    private_key.try_into().expect("Expected byteLength 32");
```

Comment:
I don't see any logic in JS that would prevent passing keys of other length. Maybe handle it gracefully and return an error?

Reply:
This is passed from the inner key handle held by JS which should be already validated. If this panics then there is a bug elsewhere during import/generation of the key.

Reply:
Ok 👍